app/services/moves/importer.rb

```diff
@@ -28,7 +28,6 @@ def import_people
       people.each do |offender_no, person_data|
         profile = People::Importer.new(person_data).call
         new_person_count += 1 if profile.new_record?
-        profile.save!
```

Comment:
Where does the save occur if we remove this line? I'm assuming there's a test which would fail if it didn't, but `profile.changed?` on L34/35 returns `false` if the record was never persisted.

Reply:
See other branch. profile.changed? actually returns true in this scenario, hence why the test doesn't fail